registrations/tests/test_utils.py

```diff
@@ -0,0 +1,76 @@
+from datetime import timedelta
+
+import pytest
+from django.utils.timezone import localtime
+
+from events.tests.factories import EventFactory, PlaceFactory
+from registrations.utils import create_event_ics_file_content
+
+
+@pytest.mark.django_db
+def test_ics_file_cannot_be_created_without_start_time():
+    event = EventFactory(start_time=None)
+    with pytest.raises(ValueError) as ex:
+        create_event_ics_file_content(event)
+    assert "Event doesn't have start_time or name. Ics file cannot be created." in str(
```

Comment:
`pytest.raises` also has the keyword argument `match` for checking if the exception string matches the given value (see https://docs.pytest.org/en/7.4.x/reference/reference.html#pytest.raises)

Reply:
This was new to me but really nice one. Changed now